kubernetes-client/package.yaml

```diff
@@ -0,0 +1,45 @@
+name: kubernetes-client
+version: 0.1.0.0
+description: |
+  This package contains functions for working with kubeconfig files.
```

Comment:
This description should get updated.

Reply:
Good catch. Done. And added a few more fields.